### PR TITLE
[WFLY-17408] Manage the plexus-utils dependency in full WildFly's own…

### PR DESCRIPTION
**Pull request by soul2zimate**
… dependencyManagement

Issue: https://issues.redhat.com/browse/WFLY-17408

related WFCORE PR: https://github.com/wildfly/wildfly-core/pull/5328

### CODE REVIEW

File: boms/common-ee/pom.xml [hunk 1]
```diff
@@ -771,6 +771,13 @@
                 <version>${version.com.fasterxml.jackson.jr.jackson-jr-objects}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.codehaus.plexus</groupId>
+                <artifactId>plexus-utils</artifactId>
+                <version>${version.org.codehaus.plexus.plexus-utils}</version>
+                <scope>test</scope>
+            </dependency>
```
Comment (bstansberry) [inline boms/common-ee/pom.xml:779]: Please move this into the root pom. This bom should only include production code dependencies. I'd like to create a testbom in full WF like we have in WF Core[1], so until we do that I'm leaving test deps in the root pom and add comments to them to note they are testsuite-only.

[1] https://issues.redhat.com/browse/WFLY-17412

File: pom.xml [hunk 2]
```diff
@@ -599,6 +599,7 @@
         <version.org.reactivestreams>1.0.4</version.org.reactivestreams>
         <version.org.syslog4j>0.9.30</version.org.syslog4j>
         <!-- Test dependency -->
+        <version.org.codehaus.plexus.plexus-utils>3.5.0</version.org.codehaus.plexus.plexus-utils>
```
Comment (bstansberry) [inline pom.xml:602]: Please move this alphabetically with the others. The 'Test dependency' comment is unneeded IMHO. Documentation of why we have a particular dependency is better placed next to the dependencyManagement entry for it.

If you can also move the version.org.keycloak.... property into the correct alphabetical position I'd appreciate it.

**Review by bstansberry — CHANGES_REQUESTED** [review]
Thanks for handling this, @soul2zimate! I have a couple comments.

File: pom.xml (hunk 2 above)
Comment (soul2zimate) [reply pom.xml]: fixed, Thanks

File: boms/common-ee/pom.xml (hunk 1 above)
Comment (soul2zimate) [reply boms/common-ee/pom.xml]: fixed, I read from pom.xml there is https://issues.redhat.com/browse/WFLY-17331 already.

**Comment by soul2zimate** [comment]
retest this please

**Comment by bstansberry** [comment]
Thank you, @soul2zimate!

**Pull request merged**